XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -72,29 +72,91 @@
             </Grid>
         </DataTemplate>
 
-        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
-            <Grid Margin="0,12,0,12">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" MinWidth="550"/>
-                </Grid.ColumnDefinitions>
-                <TextBlock Grid.Column="1"
-                           Text="{x:Bind Name}"
-                           x:Phase="1"
-                           Margin="0,12,0,0"/>
+        <DataTemplate x:Key="MessageViewTemplate" x:DataType="local1:Message">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <Grid Height="Auto" Grid.Row="1" Margin="5" HorizontalAlignment="{x:Bind MsgAlignment}">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*"/>
+                    </Grid.ColumnDefinitions>
+                    <StackPanel Height="75" Width="350" Padding="10 0 0 10" Background="{x:Bind BgColor}" CornerRadius="4">
+                        <TextBlock Text="{x:Bind MsgText}" Padding="0 10 0 0" FontSize="20" Foreground="White"></TextBlock>
+                        <TextBlock Text="{x:Bind MsgDateTime}" Padding="0 0 0 10" FontSize="15" Foreground="White"></TextBlock>
+                    </StackPanel>
+                </Grid>
             </Grid>
         </DataTemplate>
+
+        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
+            <TextBlock Text="{x:Bind Name}" x:Phase="1" Margin="0,5,0,5"/>
+        </DataTemplate>
+        
     </Page.Resources>
```

Comment:
Would be great to add more links to the Documentation section, especially to CollectionViewSource and drag-and-drop docs.